java-client/src/test/java/org/apache/pegasus/client/TestBasic.java

```diff
@@ -2780,4 +2780,103 @@ private void assertScanResult(
           new String(actuallyRes.results.get(i - startIndex).getRight()));
     }
   }
+
+  @Test // To create a timeout condition,need to change configuration/pegasus.properties timeout
+  // parameter to 1 in this case
```

Comment:
I don't see the change of this pr, so your unit test shouldn't pass?

However, you shouldn't change the config, which may be used for other unit test and cause timeout. if you want create `client` with specific parameters, you should use `createClient` api.